PluginsAndFeatures/azure-toolkit-for-intellij/azure-intellij-plugin-cosmos/src/main/java/com/microsoft/azure/toolkit/intellij/cosmos/creation/CreateCosmosDBAccountAction.java

```diff
@@ -0,0 +1,79 @@
+import com.microsoft.azure.toolkit.lib.common.operation.AzureOperation;
+import com.microsoft.azure.toolkit.lib.common.operation.OperationBundle;
+import com.microsoft.azure.toolkit.lib.common.task.AzureTaskManager;
+import com.microsoft.azure.toolkit.lib.common.utils.Utils;
+import com.microsoft.azure.toolkit.lib.cosmos.AzureCosmosService;
+import com.microsoft.azure.toolkit.lib.cosmos.CosmosDBAccountDraft;
+import com.microsoft.azure.toolkit.lib.cosmos.CosmosDBAccountModule;
+import com.microsoft.azure.toolkit.lib.cosmos.model.DatabaseAccountKind;
+import com.microsoft.azure.toolkit.lib.resource.AzureResources;
+import com.microsoft.azure.toolkit.lib.resource.ResourceGroup;
+import com.microsoft.azure.toolkit.lib.resource.task.CreateResourceGroupTask;
+import org.apache.commons.collections4.CollectionUtils;
+
+import javax.annotation.Nonnull;
+import javax.annotation.Nullable;
+import java.util.List;
+import java.util.Objects;
+
+import static com.microsoft.azure.toolkit.lib.Azure.az;
+
+public class CreateCosmosDBAccountAction {
+    public static void create(@Nonnull Project project, @Nullable CosmosDBAccountDraft.Config data) {
+        Azure.az(AzureAccount.class).account();
+        AzureTaskManager.getInstance().runLater(() -> {
+            final CosmosDBAccountCreationDialog dialog = new CosmosDBAccountCreationDialog(project);
+            if (Objects.nonNull(data)) {
+                dialog.getForm().setValue(data);
+            }
+            dialog.setOkActionListener(config -> {
+                doCreate(config, project);
+                dialog.close();
+            });
+            dialog.show();
+        });
+    }
+
+    public static CosmosDBAccountDraft.Config getDefaultConfig(final ResourceGroup resourceGroup) {
+        final List<Subscription> selectedSubscriptions = az(AzureAccount.class).account().getSelectedSubscriptions();
+        Preconditions.checkArgument(CollectionUtils.isNotEmpty(selectedSubscriptions), "There are no subscriptions in your account.");
+        final String name = String.format("cosmos-db-%s", Utils.getTimestamp());
+        final String defaultResourceGroupName = String.format("rg-%s", name);
+        final Subscription subscription = resourceGroup == null ? selectedSubscriptions.get(0) : resourceGroup.getSubscription();
+        final ResourceGroup group = resourceGroup == null ? az(AzureResources.class).groups(subscription.getId())
+                .create(defaultResourceGroupName, defaultResourceGroupName) : resourceGroup;
+        final CosmosDBAccountDraft.Config config = new CosmosDBAccountDraft.Config();
+        config.setName(name);
+        config.setSubscription(subscription);
+        config.setResourceGroup(group);
+        config.setKind(DatabaseAccountKind.SQL);
+        return config;
+    }
+
+    @AzureOperation(name = "kubernetes.create_service.service", params = {"config.getName()"}, type = AzureOperation.Type.ACTION)
```

Comment:
`kubernetes`?

Reply:
related typo has been resolved in https://github.com/microsoft/azure-tools-for-java/pull/6852/commits/3ed0f70f9241d192a82513453279ffce20d16a7b